### PR TITLE
Apply Terra-ish look-and-feel

### LINKED ISSUE #361: Update to stylized JMUI
This includes updating how the details page is structured, using new icons and methods of clicking through the UI. Moving things like comments

### PR DESCRIPTION
**Pull request by rsasch**
Job List:
job-list

Job Details:
job-details

Closes #361

### CODE REVIEW

**Comment by rexwangcc** [comment]
I'm going to list some findings in the comments :)

**Comment by rexwangcc** [comment]
When the Cromwell returns an error, the style seems to overlap with the error message
Screen Shot 2019-03-11 at 10 38 28 PM

**Comment by rexwangcc** [comment]
When you compress the window size, the top bar seems to shadow the text:
Screen Shot 2019-03-11 at 10 43 14 PM

while the old one doesn't seem to have this issue:
Screen Shot 2019-03-11 at 10 44 51 PM

**Comment by rexwangcc** [comment]
Somehow the text in the buttons on the list view page is not aligned to center on my local JMUI instance...
Screen Shot 2019-03-11 at 10 45 30 PM

File: ui/src/app/assets/images/header-left-hexs.svg [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+					<path class="st1" d="M24,67.8v8.4c0,1,0.6,1.9,1.5,2.4l7.5,4.2c0.9,0.5,2,0.5,2.9,0l7.5-4.2c0.9-0.5,1.5-1.4,1.5-2.4v-8.4
+						c0-1-0.6-1.9-1.5-2.4l-7.5-4.2c-0.9-0.5-2-0.5-2.9,0l-7.5,4.2C24.6,65.9,24,66.8,24,67.8"/>
+					<path class="st1" d="M0,67.8v8.4c0,1,0.6,1.9,1.5,2.4L9,82.8c0.9,0.5,2,0.5,2.9,0l7.5-4.2c0.9-0.5,1.5-1.4,1.5-2.4v-8.4
+						c0-1-0.6-1.9-1.5-2.4l-7.5-4.2c-0.9-0.5-2-0.5-2.9,0l-7.5,4.2C0.6,65.9,0,66.8,0,67.8"/>
+					<path class="st1" d="M84,7.8v8.4c0,1,0.6,1.9,1.5,2.4l7.5,4.2c0.9,0.5,2,0.5,2.9,0l7.5-4.2c0.9-0.5,1.5-1.4,1.5-2.4V7.8
+						c0-1-0.6-1.9-1.5-2.4l-7.5-4.2c-0.9-0.5-2-0.5-2.9,0l-7.5,4.2C84.6,5.9,84,6.8,84,7.8"/>
+					<path class="st1" d="M60,7.8v8.4c0,1,0.6,1.9,1.5,2.4l7.5,4.2c0.9,0.5,2,0.5,2.9,0l7.5-4.2c0.9-0.5,1.5-1.4,1.5-2.4V7.8
+						c0-1-0.6-1.9-1.5-2.4l-7.5-4.2c-0.9-0.5-2-0.5-2.9,0l-7.5,4.2C60.6,5.9,60,6.8,60,7.8"/>
+					<path class="st1" d="M36,7v8.4c0,1,0.6,1.9,1.5,2.4L45,22c0.9,0.5,2,0.5,2.9,0l7.5-4.2c0.9-0.5,1.5-1.4,1.5-2.4V7
+						c0-1-0.6-1.9-1.5-2.4l-7.5-4.2c-0.9-0.5-2-0.5-2.9,0l-7.5,4.2C36.6,5.1,36,6,36,7"/>
```
Comment (rexwangcc) [inline ui/src/app/assets/images/header-left-hexs.svg]: What does this do?

Comment (rsasch) [reply ui/src/app/assets/images/header-left-hexs.svg]: This is an image that is laid on top of the "Job Manager" header; see it rendered [here](https://github.com/DataBiosphere/job-manager/blob/7f059301678fa7c34ebb2d1f99ffe28cd397ef6e/ui/src/app/assets/images/header-left-hexs.svg ).

**Comment by rsasch** [comment]
@rexwangcc I wasn't able to reproduce

> When you compress the window size, the top bar seems to shadow the text:

but I added some CSS to prevent the header cells text from wrapping to a new line.
I also wasn't able to reproduce

> Somehow the text in the buttons on the list view page is not aligned to center on my local JMUI instance...

but I tweaked the CSS to hopefully make that more stable.
And I fixed the header location on the error page.

File: ui/src/app/assets/images/header-left-hexs.svg (hunk 1 above)
Comment (rexwangcc) [reply ui/src/app/assets/images/header-left-hexs.svg]: Wow, this is ..... awesome 😲

**Comment by rexwangcc** [comment]
@rsasch Thanks for the quick response!

I can see the header location on the error page is fixed now.

regarding the
> When you compress the window size, the top bar seems to shadow the text:

You could go to http://localhost:4200/jobs?q=status%3DOnHold and drag the edge of the Chrome window all the way left to make it slim, and the query builder bar will start to shadow the list view's headers.

**Comment by rsasch** [comment]
@rexwangcc I guess I'm not clear on what the preferred behavior is when the screen is made that small.  Can you elaborate?

**Comment by rexwangcc** [comment]
@rsasch I'm not sure what the preferred behavior is and having second thought now since I just found this issue on the current version of JMUI. It should be addressed in a standalone issue not this PR.

**Comment by rsasch** [comment]
Thanks for the reviewing! 🥇

**Pull request merged**